requirements.txt

```diff
@@ -4,3 +4,5 @@ matplotlib==3.0.3
 numpy==1.16.4
 scipy==1.2.1
 seaborn==0.9.0
+lxml==4.5.2
```

Comment:
If my understanding is correct, it is needed by the report_generator_test.py right?

Reply:
Yes, I believe that's correct.